Rom.py

```diff
@@ -340,7 +352,7 @@ def patch_rom(world, rom, hashtable, beep='normal', color='red', sprite=None):
                         rom.write_int16(0x15DB5 + 2 * offset, 0x0640)
                     elif room_id == 0x00d6 and world.fix_trock_exit:
                         rom.write_int16(0x15DB5 + 2 * offset, 0x0134)
-                    elif room_id == 0x000c and world.fix_gtower_exit: # fix ganons tower exit point
+                    elif room_id == 0x000c and world.fix_gtower_exit and world.mode != 'inverted': # fix ganons tower exit point
```

Comment:
Surely this could be handled by not setting world.fix_gtower_exit for inverted mode, and thus omitting that check here?